contributors/join-us/candidates.md

```diff
@@ -0,0 +1,38 @@
+
+### [Recruiter Screen](https://handbook.mattermost.com/contributors/join-us/staff-recruiting#recruiter-screen-optional)
+
+The Recruiter for each role conducts a screening interview. The screen includes standard Mattermost questions and specific questions to assess the candidate's experience, skillset, and mindset. The Recruiter also shares an overview of the [Mattermost Leadership Principles](https://handbook.mattermost.com/contributors/onboarding/staff-onboarding-guide#leadership-principles). Candidates are encouraged to ask any questions about the role, team, and Mattermost. We want you to learn about us as much as we learn about you.
+
+### Interview Panel
+
+The Hiring Manager's interview is part of the first interview panel. This panel also includes 3 - 4 team members that are either members of the direct team the potential candidate would be joining or cross-team members that this role will work closely with. Each interviewer has specific focus areas for their portion of the interview to assess one area of the candidate's experience and skillset.
+
+### Interview Project
+
+Most roles at Mattermost include a take-home project as part of the interview process. Projects are typically assigned after the second or third intervew (including Recruiter Screen).
+
+**Getting Started with the Project**
+
+All projects indicate the expected amount of time required to complete the project. This is intended to help provide each candidate with clarity on timing expectations. When you decide on the project you'd like to complete, your Recruiter and Hiring Manager will give you any additional information related to the project. Your Recruiter may ask you to sign an agreement ahead of the interview project, which will be one of the following:
+
+* The [Contributor Licence Agreement](https://mattermost.org/mattermost-contributor-agreement/) ("CLA") will be required to sign if your take-home project forms part of our [Mattermost Community](https://handbook.mattermost.com/contributors/join-us#open-source-contributors).
+* The [NDA agreement](https://na3.docusign.net/Member/PowerFormSigning.aspx?PowerFormId=75563bac-a0ac-4aef-a434-06d9c33c1f90&env=na3&acct=e5bb8810-00ab-47c2-af80-9d331c19377f&v=2) may be required if your take-home project doesn't require the CLA to be signed. Your Recruiter will confirm this step.
+
+**Gift Card Options**
+
+Mattermost recognizes and appreciates that candidates spend personal time to complete this project. To share our appreciation, candidates recieve a "Thank You" gift card upon completion of the interview project. Candidates can choose from a variety of gift cards, including: Apple iTunes, Google Plan, eBay, Visa Gift Card. If you are located in a region where you aren't able to use these gift card options, please let your Recruiter know. We're happy to work with you to identify a gift card that you can use.
+
```

Comment:
**References stage**
In between final round of interviews, we run references. We use SkillSurvey platform. Candidates should provide 3-4 references, where at least 1 should be a direct Manager.

Reply:
@natalie-hub

Reply:
This is great, thanks @teresa-novoa !